README.md

```diff
@@ -1,8 +1,15 @@
-# Commerce BTCPay
+# Accept Bitcoin on your Drupal Commerce Store using BTCPay Server
 
-This module provides a [Drupal Commerce 2.x](https://www.drupal.org/project/commerce) payment plugin for [BTCPay Server](https://docs.btcpayserver.org). This allows you to accept cryptocurrencies without a 3rd party intermediary by becoming your own payment processor.
+Introducing BTCPay Server payment module for [Drupal Commerce 2.x](https://www.drupal.org/project/commerce). Drupal Commerce Store owners can now accept payments using Bitcoin and other cryptocurrencies directly through BTCPay Server without any third-party intermediary.
 
-Visit the [project page on Drupal.org](https://drupal.org/project/commerce_btcpay)
+BTCPay Server supports a wide range of cryptocurrencies, with the potential for future extensions. Here are the currencies you can use now on BTCPay Server:
+
+- BTC (Bitcoin)
+- Bitcoin layer-two network (the lightning network) for fast and zero/low-fee transactions
```

Comment:
nitpick: looks good but I meant more that you should have capital letters "Lightning Network"